src/main/java/data_access/FirestoreUserDataAccessObject.java

```diff
@@ -121,10 +121,4 @@ public Club getClub(String email) {
         }
         return null; // Return null if no club is found
     }
-
```

Comment:
this was a bit overkill anyways. Good idea just sticking to the E-Mails as unique identifiers